harness/Adapter/index.tsx

```diff
@@ -73,21 +70,20 @@ const inputOptions = [{
 
 const exampleData = {
   mtg: mtg,
-  address: address
+  address: address,
```

Comment:
trailing comma, which may become a lint issue in the future if the default linter is implemented

Reply:
Good catch